configuration/secrets.rst

```diff
@@ -250,7 +250,7 @@ manually store this file somewhere and deploy it. There are 2 ways to do that:
    .. code-block:: terminal
 
        # this command only gets the value of the key; you must also set an env var
-       # in your system with this value (e.g. `export SYMFONY_DECRYPTION_SECRET=...`)
+       # in your system with this value (e.g. ``export SYMFONY_DECRYPTION_SECRET=...``)
```

Comment:
not sure if we need the backticks here at all as they have no practical meaning

Reply:
Oh, sure. I didn't realized this was a shell comment.
Thank you for pointing!